src/pages/home/report/ReportActionCompose.js

```diff
@@ -217,20 +218,18 @@ class ReportActionCompose extends React.Component {
                                             <CreateMenu
                                                 isVisible={this.state.isMenuVisible}
                                                 onClose={() => this.setMenuVisibility(false)}
-                                                onItemSelected={() => this.setMenuVisibility(false)}
-                                                menuItems={[
-                                                    {
-                                                        icon: Paperclip,
-                                                        text: 'Upload Photo',
-                                                        onSelected: () => {
-                                                            setTimeout(() => {
-                                                                openPicker({
-                                                                    onPicked: file => displayFileInModal({file}),
-                                                                });
-                                                            }, 10);
-                                                        },
-                                                    },
-                                                ]}
+                                                onItemSelected={(item) => {
+                                                    if (item.text === CREATE_MENU_ITEMS.ATTACHMENT.text) {
+                                                        setTimeout(() => {
+                                                            openPicker({
+                                                                onPicked: file => displayFileInModal({file}),
+                                                            });
+
+                                                            this.setMenuVisibility(false);
```

Comment:
Wouldn't we want to hide LHN for every menu item, rather than just the ATTACHMENT?

Reply:
Yep, will update
`this.setMenuVisibility(false);` - this is the Create menu modal though not LHN